### ISSUE TITLE
Please consider using redis-py

### ISSUE THREAD

**Issue by mccoy** [alexk307/redis_cache#11]
By using your own hand-rolled connection to redis you cut off using unix domain sockets and taking advantage of other features of redis.  This cache looks nice, but there is no reason for you to be making your own interface to redis.

**Comment by alexk307** [comment]
Yeah I wrote that as an exercise to be honest. With the new PR coming in (I just have to fix up some tests) you are able to specify any backend that implements an interface (therefore it will be a cache decorator for any backend, not just redis). You will be able to define the interface to use with redis-py as well.
